tap_google_ads/sync.py

```diff
@@ -24,16 +46,23 @@ def do_sync(config, catalog, resource_schema, state):
 
     core_streams = initialize_core_streams(resource_schema)
     report_streams = initialize_reports(resource_schema)
+    resuming_stream, resuming_customer = get_currently_syncing(state)
+
+    if resuming_stream:
+        selected_streams = shuffle(selected_streams, "tap_stream_id", resuming_stream)
```

Comment:
These should be sorted (not sure if that's accomplished elsewhere, but it needs to be there for consistency across tap runs since we can't trust the DB ordering to remain consistent across discovery jobs).